### PR TITLE
[stable-2.10] import_playbook - change additional params to deprecation (#72987)

### PR DESCRIPTION
**Pull request by samdoran**
##### SUMMARY

Backport of #72987 for Ansible 2.10.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`lib/ansible/modules/import_playbook.py`

### CODE REVIEW

File: test/lib/ansible_test/_data/sanity/pylint/plugins/deprecated.py [hunk 1]
```diff
@@ -233,7 +233,7 @@ def visit_call(self, node):
                     this_collection = collection_name == (self.collection_name or 'ansible.builtin')
                     if not this_collection:
                         self.add_message('wrong-collection-deprecated', node=node, args=(collection_name,))
-                else:
+                elif self.collection_name is not None:
```
Comment (relrod) [inline test/lib/ansible_test/_data/sanity/pylint/plugins/deprecated.py:236]: @samdoran is this change related? If it's intentional, it likely needs a changelog mention since `ansible-test` is a shipped product.

**Review by relrod — CHANGES_REQUESTED** [review]

File: test/lib/ansible_test/_data/sanity/pylint/plugins/deprecated.py (hunk 1 above)
Comment (relrod) [reply test/lib/ansible_test/_data/sanity/pylint/plugins/deprecated.py:236]: Ah just saw [the comment](https://github.com/ansible/ansible/pull/72987#issuecomment-745618630) in the original PR mentioning it. I still think it probably merits a changelog since it's changing an `ansible-test` component that could be used by others.

Comment (samdoran) [reply test/lib/ansible_test/_data/sanity/pylint/plugins/deprecated.py:236]: I meant for this to be included in another PR that could include a more specific changelog, but it got merged before hand. I'll add a changelog for this.
